modules/shortcuts/client/controllers/shortcuts.client.controller.js

```diff
@@ -136,12 +136,14 @@ angular.module('shortcuts').controller('ShortcutsController', ['$scope', '$state
 			if (!$scope.isFavorite(shortcut)) {
 				Favorites.save(shortcut, function(favorites) {
 					$scope.user.favorites = favorites;
+					shortcut.favorites_count += 1;
```

Comment:
Just a convention thing, we use camelCase for variables and snake case for constants. Could you update this to `favoritesCount`?